flink-staging/flink-streaming/flink-streaming-connectors/pom.xml

```diff
@@ -46,9 +53,13 @@ under the License.
 		<dependency>
 			<groupId>org.apache.kafka</groupId>
 			<artifactId>kafka_2.10</artifactId>
-			<version>0.8.2.0</version>
+			<version>${kafka.version}</version>
 			<exclusions>
```

Comment:
Mh, thats not so easy to solve. We probably have to create different maven modules for the different supported versions (if its not possible to detect and use different methods at runtime)